libs/coin-evm/src/api/gasTracker/index.ts

```diff
@@ -0,0 +1,31 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { GasOptions } from "../../types";
+import { getGasOptions as ledgerGetGasOptions } from "./ledger";
+
+type GasTrackerApi = {
+  getGasOptions: ({
+    currency,
+    shouldUseEip1559,
```

Comment:
I would rather provide the full transaction rather than something that specific. 
You can then infer whatever you want from it.
Here it would be "If transaction.type === 2 -> display=eip1559"